nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/test/java/org/apache/nifi/processor/TestSimpleProcessLogger.java

```diff
@@ -68,7 +68,7 @@ public void before() {
     @Test
     public void validateDelegateLoggerReceivesThrowableToStringOnError() {
         componentLog.error("Hello {}", e);
-        verify(logger, times(1)).error(anyString(), eq(task), eq(e.toString()), eq(e));
+        verify(logger, times(1)).error(anyString(), eq(task), anyString(), eq(e));
```

Comment:
I think these modifications are insufficient to test the added functionality.
We don't necessarily need to check the full error message in all the tests - i.e. the `anyString()` may be okay in these ones, but then we should add new tests.